dot/rpc/modules/author_integration_test.go

```diff
@@ -33,7 +35,7 @@ var testInvalidExt = []byte{1, 212, 53, 147, 199, 21, 253, 211, 28, 97, 20, 26,
 func TestMain(m *testing.M) {
 	wasmFilePaths, err := runtime.GenerateRuntimeWasmFile()
 	if err != nil {
-		log.Error("failed to generate runtime wasm file", err)
+		log.Errorf("failed to generate runtime wasm file %s", err)
```

Comment:
nit

```suggestion
		log.Errorf("failed to generate runtime wasm file: %s", err)
```

Reply:
These tests are being redone by @EclesioMeloJunior so I was just trying to get them to pass since they are being rewritten

Reply:
The plan was just to move the existing tests to integration tests, and i work on unit tests while @EclesioMeloJunior works on the integrations tests. So I'm thinking its better if I not make to many changes in these files since I think it would just cause more conflicts when the new integration tests are finished

Reply:
Makes sense!